djmoney/settings.py

```diff
@@ -32,3 +32,4 @@
 FIXER_ACCESS_KEY = getattr(settings, 'FIXER_ACCESS_KEY', None)
 BASE_CURRENCY = getattr(settings, 'BASE_CURRENCY', 'USD')
 EXCHANGE_BACKEND = getattr(settings, 'EXCHANGE_BACKEND', 'djmoney.contrib.exchange.backends.OpenExchangeRatesBackend')
+RATES_CACHE_TIMEOUT = getattr(settings, 'RATES_CACHE_TIMEOUT', 600)
```

Comment:
It's cool, but leaves a bit of a desire to start prefixing the settings var names :)